pkg/loader/compose/v3.go

```diff
@@ -192,6 +192,11 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 		serviceConfig.Command = composeServiceConfig.Entrypoint
 		serviceConfig.Args = composeServiceConfig.Command
 
+		//Handling replicas
+		if composeServiceConfig.Deploy.Replicas != nil {
```

Comment:
@cdrage thats not working as `composeServiceConfig.Deploy.Replicas` is not boolean but it's pointer